tests/Fixtures/TestBundle/Entity/Dummy.php

```diff
@@ -106,6 +106,9 @@ class Dummy
     #[ORM\ManyToMany(targetEntity: RelatedDummy::class)]
     public Collection|iterable $relatedDummies;
 
+    #[ORM\OneToMany(targetEntity: AbstractDummy::class, mappedBy: 'dummy')]
+    public Collection|iterable $abstractDummies;
+
```

Comment:
we usually try to avoid modifying existing features as it'll probably break functional tests as the answer will be different.

Reply:
you'll have more work fixing every changed response so yeah definitely, especially Dummy is used on way too many use cases.

Reply:
Fixed/reverted. I used the `DummyTableInheritance*` entities which were already utilized for exactly this purpose in the functional tests.